diagnostic.md

```diff
@@ -92,7 +121,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+The div with class specialDiv will be yellow
```

Comment:
Color will be yellow and font-family will be cursive because of the inline styling.